### DISCUSSION TITLE
Feature: Add option for quickly switch between Stereo and 5.1/7.1

### DISCUSSION THREAD

**Discussion by frandj99** [File-New-Project/EarTrumpet#1403] [Ideas]
Hello, I'm a long time user of EarTrumpet and I notice Windows 11 removed the quick switch for change Stereo/5.1/7.1 setup.
A friend of mine showed me how it is on Windows 10 and it's really more quickly with just a right click on the audio icon on taskbar, instead of the long way with the old control pannel that Windows 11 users have to do:
Playback Devices > Select the Audio Card > Configure > Set the Audio Channels > 3 more clicks for "Next"/"Finish"

 It's really useful have a quick switch for change from Stereo to 5.1 when watching movies or listening music, so I hope this could be an useful feature request that could be interesting for everyone.

<a href="https://imgbox.com/0zp0lmNx" target="_blank"><img src="https://images2.imgbox.com/a7/5b/0zp0lmNx_o.png" alt="image host"/></a>
 
 Thanks anyway for the amazin project of EarTrumpet!
 It's really better than the Windows classic taskbar menu

**Comment by riverar** [comment]
That sounds like a helpful feature indeed. Thanks for letting us know!
